controllers/error-handling-controller.js

```diff
@@ -1,6 +1,25 @@
+//400 error
+exports.handlePSQL400s = (error, request, response, next) => {
+    if (error.code === "22P02") {
+        response.status(400).send({message: "Bad request made."})
+    } else {
+        next(error);
+    }
+};
+
+// 404 error
+exports.handleCustomErrors = (error, request, response, next) => {
```

Comment:
This is a great error handler, super flexible to catch any custom errors you trigger by promise.reject!